bot/plugins/simple.py

```diff
@@ -63,6 +63,12 @@
         'GitHub recently posted a blog post about me: '
         'https://github.com/readme/stories/anthony-sottile',
     ),
+    (
+        '!schedule',
+        'Monday evenings and Saturday at noon (EST)'
+        'Check !twitter and !dicsord for more, or see the google calendar'
+        'link below the stream video',
```

Comment:
this doesn't render properly (missing spaces between words) -- also let's put it in secret commands so the help doesn't get longer

Reply:
Are the secret commands also meant to be alphabetical? It seems like it, but `!question` is above `!pre-commit`, so I could move the former below the p's in this commit too